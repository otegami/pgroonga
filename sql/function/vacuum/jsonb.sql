CREATE TABLE logs (
  message jsonb
);

CREATE INDEX pgroonga_index ON logs USING PGroonga (message);

SELECT pgroonga_table_name('pgroonga_index')
\gset old_
REINDEX INDEX pgroonga_index;

SELECT pgroonga_command('object_exist',
                        ARRAY[
                          'name', :'old_pgroonga_table_name'
                        ])::json->>1;
SELECT pgroonga_vacuum();
SELECT pgroonga_command('object_exist',
                        ARRAY[
                          'name', :'old_pgroonga_table_name'
                        ])::json->>1;

DROP TABLE logs;
