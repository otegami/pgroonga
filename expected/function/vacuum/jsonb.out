CREATE TABLE logs (
  message jsonb
);
CREATE INDEX pgroonga_index ON logs USING PGroonga (message);
SELECT pgroonga_table_name('pgroonga_index')
\gset old_
REINDEX INDEX pgroonga_index;
SELECT pgroonga_command('object_exist',
                        ARRAY[
                          'name', :'old_pgroonga_table_name'
                        ])::json->>1;
 ?column? 
----------
 true
(1 row)

SELECT pgroonga_vacuum();
 pgroonga_vacuum 
-----------------
 t
(1 row)

SELECT pgroonga_command('object_exist',
                        ARRAY[
                          'name', :'old_pgroonga_table_name'
                        ])::json->>1;
 ?column? 
----------
 false
(1 row)

DROP TABLE logs;
